frame/whitelist/src/weights.rs

```diff
@@ -62,31 +56,31 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Whitelist WhitelistedCall (r:1 w:1)
 	// Storage: Preimage StatusFor (r:1 w:1)
 	fn whitelist_call() -> Weight {
-		(22_764_000 as Weight)
+		(16_122_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(2 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 	// Storage: Whitelist WhitelistedCall (r:1 w:1)
 	// Storage: Preimage StatusFor (r:1 w:1)
 	// Storage: Preimage PreimageFor (r:0 w:1)
 	fn remove_whitelisted_call() -> Weight {
-		(24_579_000 as Weight)
+		(17_798_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(2 as Weight))
 			.saturating_add(T::DbWeight::get().writes(3 as Weight))
 	}
 	// Storage: Whitelist WhitelistedCall (r:1 w:1)
 	// Storage: Preimage PreimageFor (r:1 w:1)
 	// Storage: Preimage StatusFor (r:1 w:1)
 	fn dispatch_whitelisted_call() -> Weight {
-		(9_683_513_000 as Weight)
+		(6_696_291_000 as Weight)
```

Comment:
With a weight of 6.7s this is un-callable.  
Maybe the limits in `mock.rs` are bogus, will take a look.  
@shawntabrizi

Reply:
Just me being stupid. 1 weigth = 1/1000 ns.